### PR TITLE
feature(website): sections for new landing page

### PR DESCRIPTION
**Pull request by DarkMenacer**

### CODE REVIEW

**Comment by DarkMenacer** [comment]
@mkue @ssandino 
Worked on the quotes section. Most of it is complete but I didn't add the translations code because I have a doubt there (regarding the avatar image data). I'm waiting for now, I'll get these two reviewed and then continue working on the other sections.

File: website/src/app/[lang]/[region]/v2/(home)/(sections)/faq.tsx
Comment (mkue) [inline website/src/app/[lang]/[region]/v2/(home)/(sections)/faq.tsx:39]: Is the "text-left" required? Seems to me that this should be the default and part of the AccordionTrigger.

**Comment by mkue** [comment]
looks good to me🙌🏼

**Comment by DarkMenacer** [comment]
@mkue @ssandino 
Added two more sections
overview.tsx and approach.tsx

Could you review these sections as well?

File: shared/locales/en/website-home2.json
Comment (mkue) [inline shared/locales/en/website-home2.json:127]: Was this intended?

**Review by mkue — APPROVED** [review]
Looks good! Let's merge it and do the next sessions in a new PR, ok? This makes it a little easier to review.

**Comment by DarkMenacer** [comment]
Works

**Comment by DarkMenacer** [comment]
@mkue Can you merge this branch, I'll quickly take up the next sections...
We plan to present the new landing page during the coming townhall on the 1st of May

**Review by ssandino — APPROVED** [review]
@DarkMenacer, I think you should be able to merge this yourself, as @mkue has already reviewed it and one approval should make it mergeable for you. Please let me know if this option is somehow not available to you.

**Comment by DarkMenacer** [comment]
Okay, got it
I'll merge the PR then

**Comment by DarkMenacer** [comment]
@ssandino 
Screenshot 2024-04-23 at 07 50 44

It says this
I am not able to merge it

**Review by ssandino — APPROVED** [review]

**Comment by ssandino** [comment]
@DarkMenacer ah I know why, when things change in the UI folder @renestalder is asked an additional review. @renestalder could you click "approved" (no review required). Thanks

File: ui/src/components/accordion2.tsx
Comment (renestalder) [inline ui/src/components/accordion2.tsx:56]: Add the `export` directly to the corresponding functions/const declarations. There is no benefit of collecting them at the end of the file.

File: ui/src/components/accordion2.tsx
Comment (renestalder) [inline ui/src/components/accordion2.tsx:3]: Consider using the native `<details>` element with additional group closing handlers instead of adding a dependency on top that is probably already obsolete.

We benefit from using what is already there in HTML.

File: ui/src/components/accordion2.tsx
Comment (renestalder) [inline ui/src/components/accordion2.tsx:1]: I don't like that this is just called `accordion2`. That is not a semantically useful name and I smell that there is probably already an accordion that could have been extended instead of adding another component with a not so descriptive name.

**Review by renestalder — APPROVED** [review]
@ssandino @DarkMenacer You might wanna remove me from all code owner assignments and code review tasks. I'm so out of the loop, looking at the code I feel like you are doing anything in the code style we have in that project, but I would anyway disagree with many things because that's not the way I'd consider it best practice. My reviews only lead to many comments that are not particular helpful in this code base. I've become a very picky code reviewer as I look very closely at things like accessibility and performance. And I lead teams with focus of sustainability of code bases, that remove Tailwind and unnecessary third-party React dependencies. Nothing of that is particilar useful for Social Income at the moment.

File: ui/src/components/accordion2.tsx
Comment (ssandino) [reply ui/src/components/accordion2.tsx:1]: @renestalder es, renaming to the standard name `accordion `is planned following the migration of the new design.

File: ui/src/components/accordion2.tsx
Comment (ssandino) [reply ui/src/components/accordion2.tsx:56]: @DarkMenacer thank you for considering this for a future PR.

File: ui/src/components/accordion2.tsx
Comment (ssandino) [reply ui/src/components/accordion2.tsx:3]: @DarkMenacer thank you for considering this for a future PR.

File: ui/src/components/accordion2.tsx
Comment (mkue) [reply ui/src/components/accordion2.tsx:56]: This is definitely fine as the code directly comes from https://ui.shadcn.com/docs/components/accordion and does not need to be changed apart from the tailwind styling...

**Pull request merged**
